Docs: Customize  output directory of sveld (#68)
carbon-design-system/sveld: README.md

};
```

When building the library with Rollup, TypeScript definitions will be written to the `types` folder by default.

> If we want to customize the output folder of typescript definitions , we can customize the `typesOptions.outDir` of `sveld` as below. 

Reviewer: ```suggestion
Use the `typesOptions.outDir` option to customize the output folder. 
```

> For example, if we want the output to be in `dist` folder (instead of the default `types` folder), we can enter the following

Reviewer: ```suggestion
For example, specify the following for the output to be emitted to the `dist` folder:
```
